cmake/RunUncrustify.cmake

```diff
@@ -1,16 +0,0 @@
-# HACK: This script is invoked with "cmake -P …" as a workaround to silence uncrustify.
```

Comment:
Is this file no longer needed, because the per-file targets avoid uncrustify's noisy output (except on the first run)? That kind of thing is useful to mention in the commit message.

Reply:
yeah yeah